Code/testing_common_utils.py

```diff
@@ -14,8 +16,13 @@ def predict_on_test_data(test_data_loader, model_file):
 
     model.load_state_dict(model_state_dict)
 
+    outputs = torch.from_numpy(np.array([]))
+    target_labels = torch.from_numpy(np.array([]))
     # Classify inputs
     for images, labels in test_data_loader:
         local_images, local_labels = images.to(device, dtype=torch.float), labels.to(device, dtype=torch.float)
+        print(local_labels)
         preds = model(local_images)
-        print(preds)
+        outputs = torch.cat((outputs, preds),0)
+        target_labels = torch.cat((target_labels, local_labels),0)
+    return outputs, labels

```

Comment:
```suggestion
    return outputs, target_labels
```

I think this is what you mean